### DISCUSSION TITLE
Watchtower doesn't run when scheduled

### DISCUSSION THREAD

**Discussion by gazm2k5** [containrrr/watchtower#1883] [Q&A]
I've scheduled Watchtower to run at 5AM every Sunday. It runs the first time after initial set up, (ie it did run on Sunday 10th December) but doesn't seem to automatically schedule itself again.
Two entries:
```
time="2023-12-03T05:05:26Z" level=info msg="Scheduling first run: 2023-12-10 05:00:00 +0000 GMT"
time="2023-12-10T05:03:07Z" level=debug msg="Scheduled next run: 2023-12-17 05:00:00 +0000 GMT"
```
But it didn't run this morning 17th Dec at 5AM, and hasn't scheduled itself to run on 24th.

I'm using portainer:
![image](https://github.com/containrrr/watchtower/assets/43571579/7e121891-0ba4-42d0-aea8-a068507ac249)

**Comment by piksel** [comment]
Are there no logs past the "scheduling next run"? Perhaps the container was stopped?

**Reply by gazm2k5** [reply]
The container seems to stop immediately after finishing. I thought that was normal behaviour, portainer reports it as "Stopped for 4 days with exit code 255". I thought it might be rescheduling the container to run at system level or something.

Is it meant to run 24/7 then?

**Reply by piksel** [reply]
Yes, it cannot start itself if it's not running. It should use a minimal amount of resources while idle though.
